packages/utils/src/helpers.ts

```diff
@@ -203,9 +202,9 @@ export const formatTime = date => {
   }
 };
 
-export const customSort = (direction, field) => (a, b) => {
+export const customSort = (direction: boolean, field: string) => (a, b) => {
   if (typeof a[field] === 'string') {
-    const result = a[field].localeCompare(b[field], { sensitivity: 'case' });
+    const result = a[field].localeCompare(b[field], navigator.language, { sensitivity: 'case' });
```

Comment:
I think it's not the best idea to sort it based on browser settings. Maybe `en-US`?

Reply:
Well, `undefined` is effectively equivalent to `navigator.language` in this case. `en-US` will sort things consistently across browsers, but maybe wrong if different alphabet. I am not sure what is more important here

Reply:
`en-US` won't sort it consistent with the user expectation either if the expectation is based on how it works now 🤷 - IMO the important part is getting the case insensitivity enabled, as that was intentional in [here](https://github.com/mendersoftware/gui/pull/1297)